conjur/cli.py

```diff
@@ -290,6 +310,9 @@ def run_action(resource:str, args):
             result = client.whoami()
             print(json.dumps(result, indent=4))
 
+        elif resource == 'hostfactory':
```

Comment:
looks better if we put it under 'host' logic. it becomes easier to spot

Reply:
I don't follow

Reply:
switch the order of the `elif`s. Move this block under the host `elif` block for readability